### PR TITLE
Flpath 189: Add WorkStatus to WorkFlowResponseDTO and set it before return to user

### PR DESCRIPTION
**Pull request by nirarg**

### CODE REVIEW

File: workflow-examples/run_examples.sh [hunk 1]
```diff
@@ -109,6 +109,31 @@ wait_project_start() {
...
+    -H 'Authorization: Basic dGVzdDp0ZXN0' \
+    -H "X-XSRF-TOKEN: ${TOKEN}" \
+    -b $COOKIEFP \
+    -H 'Content-Type: application/json' \
+    -d "$params")
+
+    status=$(echo "$response" | jq -r '.workStatus')
+    if [[ "$status" == "FAILED" ]]; then
+      if "$exitOnFailure"; then
+        echo_red "Error: HTTP request failed" >&2
```
Comment (eloycoto) [inline workflow-examples/run_examples.sh:127]: I would use @fail function here tbh

Comment (nirarg) [reply workflow-examples/run_examples.sh]: Done

**Pull request merged**